### DISCUSSION TITLE
Quasar2 ssr/expressjs api reload problem

### DISCUSSION THREAD

**Discussion by mirkofisic** [quasarframework/quasar#12854] [CLI - SSR mode]
I use SSR mode in which I set ssr middleware, I use app (Expressjs to create my API, app.get, app.post) and everything is ok but if in some request handler eg: app.post ('/api/user/ create ', (req, res, next) => { // whatever I chnaged }) change anything, quasar automatically refreshes the entire application but something breaks on the client side and every time I have to stop the server and run again **quasar dev -m ssr** and then it works again until the next change inside ssr middlware:

![image](https://user-images.githubusercontent.com/6698925/159128678-0575382c-9047-4ef7-802a-240583409380.png)

here is terminal log:
![image](https://user-images.githubusercontent.com/6698925/159128718-ac45f23b-c796-49b9-ad2e-bd841e5ef09c.png)

I don't get any error, does anyone have any idea where the problem is here?

**Comment by ajmas** [comment]
Have you looked at the errors and the content of the files being served up, from the browser?

**Reply by mirkofisic** [reply]
Problem is that all files content is actually html not JavaScript but why?
![image](https://user-images.githubusercontent.com/6698925/159166814-cb996fee-e203-47b5-9c13-1b61eded85e4.png)

**Comment by ajmas** [comment]
I suspect that index.html is being served up for all requests. Are you using nginx, apache or how is your server configured to serve up your files? More information the better.

**Reply by mirkofisic** [reply]
I dont use apache, nginx this is development mode and I use
Visual Studio Code
OS Windows 10
NodeJS version 16.14.0

**Reply by mirkofisic** [reply]
@ajmas I open bug issue and described in a little more detail what exactly is happening and how to reproduce the problem, please if you can take a look:
https://github.com/quasarframework/quasar/issues/12870
